### DISCUSSION TITLE
Reducing amount of updates - requiring latest version to be stable for x days?

### DISCUSSION THREAD

**Discussion by adriandmitroca** [renovatebot/renovate#16157] [Request Help] [answered]
Is there any way to require packages to not be updated (in other words - be stable) since x days?

I know we have `stabilityDays` option, but this simply delays updates by 3 days, which still for example can create 3 updates for minor patches and so on.

In ideal world (at least in our use case) would be to reduce amount of updates by ignoring all minor versions and spam, unless latest release doesn't change for specified amount of time.

**Comment by rarkins** [comment, marked as answer]
Some packages like AWS ones or Renovate itself release some regularly that you'd never reach your definition of "stability"

**Reply by adriandmitroca** [reply]
Of course, that makes sense. But there are also some potential scenarios where this definition of "stability" could make sense.

**Reply by HonkingGoose** [reply]
@adriandmitroca, I think you may want to subscribe to this issue:

- https://github.com/renovatebot/renovate/issues/12914
